src/wrappers/themis/Obj-C/objcthemis/scell_seal.h

```diff
@@ -14,16 +14,66 @@
 * limitations under the License.
 */
 
+/**
+ * @file objthemis/scell_context_imprint.h
+ * @brief secure cell seal mode interface
+ */
+
 #import <Foundation/Foundation.h>
 #import <themis/themis.h>
 #import <objcthemis/scell.h>
 
+/**
+ * @addtogroup WRAPPERS
+ * @{
+ * @addtogroup OBJC
+ * @{
+ */
+
+/** @brief Secure Cell Seal mode interface */
 @interface SCell_seal : SCell
 
+/**
+ * @brief Initialize Secure cell object in seal mode
```

Comment:
in a perfect world, as user i would like to read some info about what is seal mode. maybe, copy few lines from wiki?